config/build.js

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+
```

Comment:
This defines how resources are put together... 
There are a few improvements we could do here.